app/services/hyrax/abstract_message_service.rb

```diff
@@ -1,5 +1,5 @@
 module Hyrax
-  class MessageUserService
+  class AbstractMessageService
```

Comment:
Rename this to better reflect intent -- else, this may be easily confused with the new `MessengerService`

Reply:
Cool